socorro/unittest/external/es/test_correlations.py

```diff
@@ -53,3 +124,90 @@ def test_create_correlations_index(self):
 
         # We should be able to create that index again without any errors.
         correlations.create_correlations_index(es_index)
+
+
+class TestCoreCounts(BaseTestCorrelations):
+
+    def test_store(self):
+        correlations = CoreCounts(config=self.config)
+        today = datetime.datetime.utcnow()
+        correlations.store(
+            sample_core_counts_summary,
+            today.strftime('%Y%m%d'),
+            'core-counts',
+            'Firefox_43.0.1'
+        )
+        es_index = correlations.get_index_for_date(today.date())
+        correlations.create_correlations_index(es_index)
+        self.indices_for_deletion.append(es_index)
+        self.refresh_index(es_index)
+        result = self.es_client.search(
+            index=es_index,
+            doc_type='correlations',
+        )['hits']
+        eq_(result['total'], 1)
+        hit, = result['hits']
+        _source = hit['_source']
+        eq_(
+            _source['count'],
+            sample_core_counts_summary['Windows NT']['count']
+        )
+        eq_(
+            _source['signature'],
+            sample_core_counts_summary['Windows NT']['signatures'].keys()[0]
+        )
+        eq_(_source['date'], today.strftime('%Y-%m-%d'))
+        eq_(_source['notes'], sample_core_counts_summary['notes'])
+        eq_(_source['key'], 'core-counts')
+        eq_(_source['platform'], 'Windows NT')
+        eq_(_source['product'], 'Firefox')
+        eq_(_source['version'], '43.0.1')
+        payload, = (
+            sample_core_counts_summary['Windows NT']['signatures'].values()
+        )
+        eq_(_source['payload'], json.dumps(payload))
```

Comment:
Well, I think you could add line breaks so that there are more logical blocks. Unit tests are not fun to read, so I think it's extra important to make them as readable as possible.